modules/assim.batch/R/helpers.R

```diff
@@ -0,0 +1,30 @@
+stop <- function(...) {
```

Comment:
So my rationale for adding these is precisely to remove explicit dependencies on utils if all we need from there is the logging functions. This is particularly the case for PEcAnRTM because it almost always runs offline of the main PEcAn workflow.

Another option is to just rename all the logger functions to their R equivalents and change everything throughout PEcAn -- that way, nothing would _have_ to depend on utils for logging, but would have the desired behavior if the PEcAn logging functions were loaded.